projects/plugins/jetpack/modules/carousel/jetpack-carousel.js

```diff
@@ -1602,10 +1600,50 @@
 					}, 150 );
 				}
 			} );
+			var currentSlideObj = carousel.slides[ options.startIndex ];
```

Comment:
Just checked what's happening with `options.startIndex` when clicking between images in the image gallery before the carousel is opened. In that case, `options.startIndex` is `-1`. Would it work if we updated this line to something like the following to ensure we grab the first slide if one isn't specified?:

```suggestion
			var currentSlideObj = options.startIndex > 0 ? carousel.slides[ options.startIndex ] : carousel.slides[ 0 ];
```